cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -622,6 +624,20 @@ func (s *Server) handleReIndex(w http.ResponseWriter, r *http.Request) {
 	_ = repoTmpl.Execute(w, data)
 }
 
+func handleHealthCheck(w http.ResponseWriter, r *http.Request) {
+	switch r.Header.Get("Accept") {
+	case "application/json":
+		w.Header().Set("content", "application/json")
```

Comment:
Are you sure that the proper header name is "content" as opposed to "content-type"

Reply:
Oops